src/app/models/media-wall-custom.ts

```diff
@@ -0,0 +1,69 @@
+export interface WallHeader {
+	backgroundColor: string;
+	fontColor: string;
+}
+
+export interface WallBackground {
+	backgroundColor: string;
+}
+
+export interface WallGrid {
+	columnCount: string;
+	columnGap: string;
+}
+
+export interface WallCard {
+	fontSize: string;
+	fontColor: string;
+	backgroundColor: string;
+	accentColor: string;
+}
+
+/**
+ * @function defaultColumnCount
+ * Checks the current screen size and returns the default column count
+ *
+ * @return string
+ */
+
+export function defaultColumnCount(): string {
```

Comment:
Why such approach?
Why aren't you simply depending on flexbox this is not good approach, laying out UI by JS.